README.rst

```diff
@@ -96,9 +96,11 @@ Looking for more help? Check out our `support channels`_.
 
 .. _command-line interface: https://en.wikipedia.org/wiki/Command-line_interface
 .. _contribution docs: CONTRIBUTING.rst
-.. _Crate.io: https://crate.io/
+.. _Crate.io: https://cratedb.com/
```

Comment:
@amotl this one stays?

Reply:
**io** for the company and **db** for the product right?

Reply:
Yeah, and the statement "cratedb.com is the website of Crate.io" is also true, and correct, no?

Reply:
ah yes, good point, imprint confirms this 😄